### DISCUSSION TITLE
viewer组件属性camera【optional 指定初始化场景相机位置，默认在中国】的格式示例是什么样的

### DISCUSSION THREAD

**Discussion by kfcvs** [zouyaoji/vue-cesium#197] [Q&A] [answered]
在指南里搜索了一下没有找到相关示例

**Comment by zouyaoji** [comment, marked as answer]
文档还不完善哈，不好意思。j建议是在 tsconfig或者jsconfig配置vue-cesium的类型声明，是可以通过vscode的插件看到各组件参数的类型声明的。

https://zouyaoji.top/vue-cesium/#/zh-CN/component/quickstart#typescript-ji-dai-ma-zhi-neng-ti-shi-zhi-chi

![image](https://user-images.githubusercontent.com/24430116/165654789-f9f3df93-ed61-4f8b-8c19-4cb703be8755.png)
![image](https://user-images.githubusercontent.com/24430116/165654811-ebbeaac5-4479-4ff3-ba80-875535dd1224.png)
